plugins/builddep.py

```diff
@@ -21,38 +21,33 @@
 
 from __future__ import absolute_import
 from __future__ import unicode_literals
-from dnfpluginscore import _, logger, rpm_logger
 
 import argparse
+import logging
+import os
+import rpm
+
 import dnf
 import dnf.cli
 import dnf.exceptions
 import dnf.rpm.transaction
-import functools
-import logging
-import os
-import rpm
 
+from dnfpluginscore import _, logger, rpm_logger
```

Comment:
The rearrangement of imports should go into a separate commit. e.g. imagine needing to revert this in the future and someone else adding/removing imports in between. Not only you'd revert the rearrangement unnecessarily, you'd also have an unnecessary conflict on the revert...